int32 id
string label
sensor_msgs/PointCloud2 cloud

